packages/ui5-proxy-middleware/src/base/manifest/sapOvp.ts

```diff
@@ -0,0 +1,435 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
```

Comment:
Can we document where the schema is coming from and how/when we might want to update this generated file?

Reply:
Yup, will add it to the README